src/media/js/main.js

```diff
@@ -144,7 +144,9 @@ require(
 
     window.addEventListener(
         'resize',
-        _.debounce(function() {z.doc.trigger('saferesize');}, 200),
+        _.debounce(function() {
+            document.dispatchEvent(new CustomEvent('saferesize'));
```

Comment:
It's polyfilled in mcm so it should be fine, https://github.com/mozilla/marketplace-core-modules/blob/4d0cbe586999f84d5d83f1ab37895095b79c2b3e/core/polyfill.js#L21-L27.